delve/torch_utils.py

```diff
@@ -2,8 +2,22 @@
 
 
 class TorchCovarianceMatrix(object):
+    r"""Computes covariance matrix of features as described in https://arxiv.org/pdf/2006.08679.pdf:
+    
```

Comment:
Maybe mention also that the computation always happens with float64 values by default, regardless of the precision of the data-tensors.
This seems an unimportant technical detail, but these rounding errors compound rapidly when dealing with convolutional layers and cause massive numeric instability.

Reply:
Done!